pinot-core/src/main/java/org/apache/pinot/core/query/reduce/DistinctDataTableReducer.java

```diff
@@ -102,6 +114,19 @@ public void reduceAndSetResults(String tableName, DataSchema dataSchema, Map<Ser
     // finish the merging, sort (if ORDER BY), get iterator
```

Comment:
This DistinctTable will have the schema and thus both column names and types.

Reply:
done